lib/defaults/table.json

```diff
@@ -0,0 +1,271 @@
+[
+  {
+    "@id": "https://atomicdata.dev/properties/max",
+    "https://atomicdata.dev/properties/datatype": "https://atomicdata.dev/datatypes/integer",
+    "https://atomicdata.dev/properties/description": "The maximum size of the value",
+    "https://atomicdata.dev/properties/isA": [
+      "https://atomicdata.dev/classes/Property"
+    ],
+    "https://atomicdata.dev/properties/parent": "https://atomicdata.dev/properties",
+    "https://atomicdata.dev/properties/shortname": "max"
+  },
+  {
+    "@id": "https://atomicdata.dev/properties/color",
+    "https://atomicdata.dev/properties/datatype": "https://atomicdata.dev/datatypes/string",
+    "https://atomicdata.dev/properties/description": "The color of something.",
+    "https://atomicdata.dev/properties/isA": [
+      "https://atomicdata.dev/classes/Property"
+    ],
+    "https://atomicdata.dev/properties/parent": "https://atomicdata.dev/properties",
+    "https://atomicdata.dev/properties/shortname": "color"
+  },
+  {
+    "@id": "https://atomicdata.dev/classes/FormattedNumber",
+    "https://atomicdata.dev/properties/description": "A number that should be displayed in a certain way.",
+    "https://atomicdata.dev/properties/isA": [
+      "https://atomicdata.dev/classes/Class"
+    ],
+    "https://atomicdata.dev/properties/parent": "https://atomicdata.dev/classes",
+    "https://atomicdata.dev/properties/recommends": [
+      "https://atomicdata.dev/properties/decimalPlaces"
+    ],
+    "https://atomicdata.dev/properties/requires": [
+      "https://atomicdata.dev/properties/numberFormatting"
+    ],
+    "https://atomicdata.dev/properties/shortname": "formatted-number"
+  },
+  {
+    "@id": "https://atomicdata.dev/classes/DateFormat",
+    "https://atomicdata.dev/properties/description": "The way the date value should be displayed.",
+    "https://atomicdata.dev/properties/isA": [
+      "https://atomicdata.dev/classes/Class"
+    ],
+    "https://atomicdata.dev/properties/parent": "https://atomicdata.dev/classes",
+    "https://atomicdata.dev/properties/recommends": [],
+    "https://atomicdata.dev/properties/requires": [
+      "https://atomicdata.dev/properties/shortname"
+    ],
+    "https://atomicdata.dev/properties/shortname": "date-format",
+    "https://atomicdata.dev/properties/subresources": [
```

Comment:
I think this is useful, but maybe it should be a different relationship than subresource. Not sure.